.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+name: brew pr-pull
+
+on:
+  pull_request_target:
+    types:
+      - labeled
```

Comment:
So is this opening a PR to homebrew whenever someone adds the label to a PR? How does that work when someone doesn't know what the label does and adds it to a PR accidentally and overrides the latest version? Or how do we track which changes are currently pushed up to homebrew?

Reply:
Hey @nwneisen , not exactly. This is a workflow that triggers when a PR is created with label `pr-pull`. It runs some tests on the PR and if everything passes it is merged and the branch is deleted. The changes are not pushed to homebrew, they are within this repository and homebrew stores it as a `tap` locally. Users can run `homebrew update` which automatically pulls all the latest changes from the locally configured taps.

I'm planning to update the `mke-releases` github actions to create a PR in this repo whenever a new mke version is GA - and this action will run tests then automatically update the homebrew tap.

It's essentially copied from the k0s homebrew tap, you can see it in action there: https://github.com/k0sproject/homebrew-tap/pulls?q=

Reply:
Thanks for clarifying. I thought this might be pushing up changes to homebrew. 

So this is more of a trigger to run the tests and merge in the changes to main if they pass?

Reply:
@nwneisen Correct. We don't need to push anything up to homebrew with the homebrew tap implementation ( that is only needed if we were adding it as a default formula (i.e in [homebrew-core](https://github.com/Homebrew/homebrew-core))